azurerm/resource_arm_batch_pool.go

```diff
@@ -55,6 +55,11 @@ func resourceArmBatchPool() *schema.Resource {
 				ForceNew:         true,
 				DiffSuppressFunc: suppress.CaseDifference,
 			},
+			"max_tasks_per_node": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Default:  1,
```

Comment:
Can you add a validation for this field ?

Reply:
Done